prac_07/squaring.kv

```diff
@@ -1,15 +1,20 @@
 BoxLayout:
+    orientation: 'horizontal'
+    BoxLayout:
     orientation: 'vertical'
-    TextInput:
-        id: input_number
-        text: '7'
-        font_size: 48
-        multiline: False
-    Button:
-        text: 'Square'
-        # the following line specifies the function in the app class to call when the button is pressed
-        on_press: app.handle_calculate(int(input_number.text))
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(int(input_number.text))
+        Label:
+            id: output_label
+            font_size: 48
+            color: (255, 192, 1203, 1)
     Label:
-        id: output_label
-        font_size: 48
-        color: (0, 1, 1, 1)
+        orientation: 'vertical'
```

Comment:
You may not need this line as the Label is already oriented by line 1 `BoxLayout`?